src/payment/CreditCard.java

```diff
@@ -0,0 +1,57 @@
+package payment;
+
+public class CreditCard {
+    private String cardNumber;
```

Comment:
```suggestion
public class CreditCard {

    private String cardNumber;
```
please add an empty line between the class declaration and first method/variable

Reply:
Corrected formatting